### ISSUE TITLE
Solved: Configure access to Web GUI of Syncthing from another host in SyncTrayzor

### ISSUE THREAD

**Issue by maelcum** [canton7/SyncTrayzor#295]
Hello.

tl;dr:
Are there known problems of accessing the Syncthing GUI from another host when running Syncthing via SyncTrayzor?

Long Story:
SyncTrayzor.exe portable version 1.1.11.0
Syncthing.exe version 0.13.10
Windows 10 version Prio N 1511 build 10586.420

I had Syncthing running without SyncTrayzor and could access the GUI from other hosts. 
Syncthing is running on port 8090, the settings in Syncthing allow access from 0.0.0.0:8090, the (windows) firewall of the host Syncthing is running on has a rule to allow incoming traffic (any port, any IP) for syncthing.exe.

After installing SyncTrayzor the same Syncthing configuration is used. SyncTrayzors settings read:
` <SyncthingAddress>localhost:8090</SyncthingAddress>`
SyncTrayzor starts without problems or errors. Syncthing starts without problems or errors.
Output from Syncthing console is
```
[DJ3IB] 23:29:58 INFO: TCP listener ([::]:22000) starting
[DJ3IB] 23:29:58 INFO: Device C3EID4T is "DE71364-CL-M4" at [dynamic]
[DJ3IB] 23:29:58 INFO: Device DJ3IBA6 is "DE71364-SV-W2" at [dynamic]
[DJ3IB] 23:29:58 INFO: Device 3WA5ZSH is "DE71364-CL-M3" at [dynamic]
[DJ3IB] 23:29:58 INFO: GUI and API listening on 127.0.0.1:8090
[DJ3IB] 23:29:58 INFO: Access the GUI via the following URL: https://localhost:8090/
```
The display of the Syncthing GUI in the SyncTrayzor windows works nicely. Opening the GUI with another browser at localhost:8090 works nicely. Other hosts can communicate and sync with Syncthing.
But accessing the GUI by the hosts IP (say: 192.168.254.2:8090) does not work - there seems to be no service at this port.

The windows firewall has been turned off to make sure it is not blocking anything.
The connection has been tried from different hosts with different os'ses and different browsers to make sure its not just coincidence.
The connection has been tried with IPv4 as well as IPv6 - though I am not that great an expert and might have done everything wrong.
The synctryzor.exe.config was edited and < CefRemoteDebuggingPort > set to 2004. Afterwards Chrome was opened with http://localhost:2004 and the link klicked - unfortunately the windows is empty (got all that from here: https://github.com/canton7/SyncTrayzor/issues/262 while looking for solutions).

Nothing helped.

Just to verify, SyncTrayzor was stopped again, Syncthing started from the cmd and yes, the GUI is available again from different hosts. Syncthing stoppen, SyncTrayzor started (and with it Syncthing) ... aaaaaand its gone. The GUI.

Is there a setting that I've overlooked in SyncTrayzor or something additional that I'd need to configure? Has anybody seen this before?

Thanks for reading. Have a nice day.

**Comment by canton7** [comment]
There's an option in settings -> Syncthing which controls what address syncthing listens on, and overrides the guy listen address in syncthing's settings, otherwise there are no gotchas. Change that setting to something other than localhost and you should be fine.

**Comment by maelcum** [comment]
Hey, thank you.

I don't see why a Syncthing setting should make a difference. As already said, Syncthing is working fine without SyncTrayzor. But I'll try happily everything.
I assume you mean the GUI listen address, which has already been set to 0.0.0.0:8090 (see original post). Or is it one of the Extended Settings?
I've attached a screenshot of the settings: Could you point out what to change in addition to those?
<img width="701" alt="syncthing-settings" src="https://cloud.githubusercontent.com/assets/126681/16897768/7faf7ca6-4bbb-11e6-9a1f-ed5da4737cd3.png">
<img width="710" alt="syncthing-advanced-settings" src="https://cloud.githubusercontent.com/assets/126681/16897770/8347898a-4bbb-11e6-82e7-79d77ebefad9.png">

**Comment by canton7** [comment]
In SyncTrayzor (not Syncthing)  go to File, Settings, Syncthing. The Listen Address setting (or whatever it's called: I'm not near a computer)  overrides the listen address you set in Syncthing, when you start Syncthing within SyncTrayzor. Set that to 0.0.0.0:whatever

**Comment by maelcum** [comment]
\o/  YES! Thanks! Perfect!

**Comment by sjehuda** [comment]
Please add this command at the top of your initial post.
```
syncthing cli config gui raw-address set 0.0.0.0:8384
```
From https://forum.syncthing.net/t/accessing-gui-from-an-external-ip/22113/2

**Comment by sjehuda** [comment]
syncthing cli config gui raw-address set 0.0.0.0:8090
